packages/test/test-service-load/package.json

```diff
@@ -27,7 +27,7 @@
     "lint": "npm run eslint",
     "lint:fix": "npm run eslint:fix",
     "start": "node ./dist/nodeStressTest.js",
-    "start:mini": "node ./dist/nodeStressTest.js --profile mini",
+    "start:mini": "node ./dist/nodeStressTest.js --driver local --profile mini",
```

Comment:
local will not work here. it should be tinylicious

Reply:
oops it was just for testing